.travis.yml

```diff
@@ -12,7 +12,7 @@ env:
     - ros_release_name: kinetic
     - ubuntu_version_name: xenial
     - used_modules: check_build #check_cache,code_style_check,check_deb_make,codecov_tool
-    - remote_shell_script: 'https://raw.githubusercontent.com/shadow-robot/sr-build-tools/$toolset_branch/bin/sr-run-ci-build.sh'
+    - remote_shell_script: 'https://raw.githubusercontent.com/tum-phoenix/drive_ros_config/travis-build-script/.travis-build.sh'
```

Comment:
Uhm, I mean you could just run this script directly instead of `curl`-ing it from GitHub.